### DISCUSSION TITLE
Data Ingest Seemingly broken

### DISCUSSION THREAD

**Discussion by ow111** [Security-Onion-Solutions/securityonion#2717] [Unsupported Versions]
Been having major issues with SO showing no data ingest in the kibana dashboard from filebeat/auditbeat/anything at all since running so-features-enabled on the manager node. Running a distributed deployment installed via the ISO. 
`[2021-01-20T11:49:39,424][INFO ][logstash.outputs.elasticsearch] Elasticsearch pool URLs updated {:changes=>{:removed=>[], :added=>[https://x.x.x.x:9200/]}} [2021-01-20T11:49:39,737][ERROR][logstash.javapipeline ] Pipeline error {:pipeline_id=>"search", :exception=>#<Manticore::UnknownException: Unsupported or unrecognized`  

We were running elasticsearch over HTTPS on the searchnodes before enabling the security features so this may have caused an issue. 
Unsure what the problem is here, can give more details as required to help diagnose and maybe reproduce this issue.

**Comment by weslambert** [comment]
What version of Security Onion are you running?

**Reply by ow111** [reply]
2.3.21 at the moment

**Reply by weslambert** [reply]
Ok, we are attempting to replicate this now.

**Comment by weslambert** [comment]
So, it looks like you'll need to adjust the outputs for Logstash to allow HTTP communication, by copying the various outputs for zeek/suricata/etc in `/opt/so/saltstack/default/salt/logstash/pipelines/config/so/` to the equivalent `/opt/so/saltstack/local/salt/logstash/pipelines/config/so` directory, modifying `https` -> `http` in the Elasticsearch output, then restart Logstash on each node with `salt "*" cmd.run "so-logstash-restart"`.  This issue should be resolved in the next resolve, but may be handled in a different manner at that time.

**Reply by ow111** [reply]
Thanks very much Wes, I will give this a try.

**Reply by thunderwood19** [reply]
Did this work? Been banging my head against a wall for a week now over this. 

Edit:
@weslambert your answer is a little confusing, what would the command look like for this/what conf files?

**Reply by weslambert** [reply]
For example, copy `/opt/so/saltstack/default/salt/logstash/pipelines/config/so/9000_output_zeek.conf.jinja` to `/opt/so/saltstack/local/salt/logstash/pipelines/config/so/9000_output_zeek.conf.jinja`, and modify the Elasticsearch output to use http vs https.

**Comment by thunderwood19** [comment]
so-features-enable caused this for me.  Changing it (on master) in /opt/so/saltstack/local/pillar/global.sls from true to false.

**Reply by weslambert** [reply]
It's because it's trying to send to attempt TLS for a connection to a host that is not using TLS. You'll need to change the outputs as mentioned above.
